migration/idempotent/010-telemetry.sql

```diff
@@ -85,7 +85,7 @@ $$
         SELECT count(*)::TEXT INTO result FROM _prom_catalog.label WHERE key = '__tenant__';
         PERFORM _ps_catalog.apply_telemetry('metrics_multi_tenancy_tenant_count', result);
 
-        SELECT count(*)::TEXT INTO result FROM _prom_catalog.label_key WHERE key = '__cluster__';
+        SELECT count(*)::TEXT INTO result FROM _prom_catalog.label_key WHERE key = 'cluster';
```

Comment:
Thanks @JamesGuthrie , I'm trying to understand the purpose of `promscale_metrics_ha_cluster_count` . It makes sense to use  `_prom_catalog.label` if we are trying to find number of prometheus clusters remote writing into DB(e.g. 2 k8s clusters remote writing into single timescale), otherwise `_prom_catalog.label_key` might be enough.

Probably @VineethReddy02 might able to clarify.

Reply:
I was wondering the same thing. I had to assume based on the name and code what it's trying to do. @Harkishen-Singh you implemented this initially, do you know?

Reply:
Yes, the aim of `promscale_metrics_ha_cluster_count` is to know how many `HA systems` are being used in some Promscale instance. So, this should be `label` table. I think label key is just the key, so the count will always be one.

Reply:
Good catch!

Reply:
Thanks @Harkishen-Singh & @JamesGuthrie. Applied the suggestion :)